pipeline/workflow-builder.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/kiegroup/kogito-swf-builder-nightly:main-2024-02-13 AS builder
+FROM quay.io/kiegroup/kogito-swf-builder-nightly:main-2024-02-19 AS builder
```

Comment:
@gabriel-farache do you think this image may contain the needed fix to test?

Reply:
@masayag which fix? The 500 errors are on the DI index side so https://quay.io/repository/kiegroup/kogito-data-index-ephemeral-nightly (in case of ephemeral)

Reply:
yes, I missed the fact the fix was into the DI.